src/components/Profile/DeleteAccountButton.tsx

```diff
@@ -33,8 +31,6 @@ const DeleteAccountButton = ({ isDisabled }: DeleteAccountButtonProps) => {
   const onDeleteConfirmed = () => {
     closeModal();
     deleteAccount()
-      .then(() => logoutUser())
-      .then(() => mutate(makeUserProfileUrl()))
```

Comment:
I think we still need to mutate the profile @osamasayed ?

to clear out the previous data?